src/it/scala/CirceDebeziumSpec.scala

```diff
@@ -0,0 +1,222 @@
+package compstak.circe.debezium
+
+import scala.concurrent.duration._
+import scala.util.Random
+
+import cats.effect.testing.scalatest.AsyncIOSpec
+import cats.effect.testing.scalatest.scalacheck.EffectCheckerAsserting
+
+import cats.effect.{IO, Sync}
+
+import org.scalatest._
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.scalacheck.{CheckerAsserting, ScalaCheckPropertyChecks}
+
+import org.testcontainers.containers.Network
+import com.dimafeng.testcontainers.{ ForAllTestContainer, GenericContainer, KafkaContainer, MultipleContainers, PostgreSQLContainer }
+import GenericContainer.DockerImage
+
+import doobie._
+import doobie.implicits._
+import cats._
+import cats.data._
+import cats.effect._
+import cats.implicits._
+
+import org.scalacheck._
+import org.scalacheck.ScalacheckShapeless._
+import org.scalacheck.cats.implicits._
+
+import org.flywaydb.core.Flyway
+
+import org.http4s._, Method._, client.Client, client.asynchttpclient.AsyncHttpClient, client.dsl.io._
+import org.http4s.circe.CirceEntityCodec._
+
+import io.circe._, literal._
+import io.circe.parser._
+
+import fs2.kafka._
+
+class CirceDebeziumSpec
+  extends AsyncIOSpec
+  with Matchers
+  with BeforeAndAfterAll
+  with ScalaCheckPropertyChecks
+  with ForAllTestContainer
+  with scalatest.IOChecker {
+
+  
+  case class Debezium(field1: Int, field2: Short, field3: Long, field4: Boolean, field5: String, field6: Float, field7: Double, field8: Option[Int])
+
+  implicit val dbDecoder = Decoder.forProduct8("field1", "field2", "field3", "field4", "field5", "field6", "field7", "field8")(Debezium.apply)
+
+  implicit val intKeyDeserializer = Deserializer.lift { bytes =>
+    IO.fromEither(decode[DebeziumKey[Int]](new String(bytes, "UTF8")))
+  }
+
+  implicit val dbvValueDeserializer = Deserializer.lift { bytes =>
+    IO.fromEither(decode[DebeziumValue[Debezium]](new String(bytes, "UTF8")))
+  }
```

Comment:
I don't quite understand why you need to do `new String(bytes, "UTF8")`, in the consumers I wrote we got the Json string straight from Kafka. What am I missing?